noxfile.py

```diff
@@ -32,17 +32,19 @@ def _expand_env_vars(file_path: Path):
 
 
 @nox.session(python=["3.8", "3.9", "3.10"])
-@nox.parametrize("airflow", ["2.4", "2.5"])
+@nox.parametrize("airflow", ["2.2.4", "2.3", "2.4", "2.5"])
 def test(session: nox.Session, airflow) -> None:
     """Run both unit and integration tests."""
-    session.run("echo", "$PWD")
     env = {
         "AIRFLOW_HOME": f"~/airflow-{airflow}-python-{session.python}",
         "AIRFLOW__CORE__ALLOWED_DESERIALIZATION_CLASSES": "airflow\\.* astro\\.* cosmos\\.*",
     }
 
-    session.install(f"apache-airflow=={airflow}")
     session.install("-e", ".[all,tests]")
+    session.install(f"apache-airflow=={airflow}")
+
+    if airflow in ("2.2.4", "2.3", "2.4"):
```

Comment:
Leave a comment here mentioning why we no longer need this 2.5+. My hope is in the future as we no longer support these versions we can remove these lines.